json_schema/api/operation.yaml

```diff
@@ -0,0 +1,32 @@
+Operation:
+  description: The status of a long running operation.
+  properties:
+    path:
+      type: string
+      description:
+        The server-assigned path, which is only unique within the same
+        service that originally returns it.
+    done:
+      type: boolean
+      description: >-
+        If the value is false, it means the operation is still in progress.
+        If true, the operation is completed, and either response or error
+        is available.
+    error:
+      $ref: '#/components/schemas/Error'
```

Comment:
this doesn't resolve to anything and you don't have the schema defined with the same language as the markdown description. `Error` and `response` are mutually exclusive in the text, but not here.